File: jax-rs-models/src/main/java/com/untitleddoc/cadencecalc/jaxrs/models/Perimeter.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.untitleddoc.cadencecalc.jaxrs.models;

import java.io.Serializable;
import javax.xml.bind.annotation.XmlRootElement;
import lombok.AllArgsConstructor;
import lombok.Data;

/**
 *
 * @author Novikov
 */
@XmlRootElement
@Data
@AllArgsConstructor
public class Perimeter implements Serializable {

    private static final long serialVersionUID = 165573926447503641L;

    private String etrto;
    private String french;
    private int perimeterValue;

    public Perimeter() {
    }

    public String displayValue() {
        return getEtrto() + " (" + getFrench() + ")";
    }
}

File: jax-rs-models-kotlin/src/main/kotlin/com/untitleddoc/cadencecalc/jaxrs/models/kotlin/Perimeter.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
/*

  @author k-aki
  Created on 2018/06/03
*/

package com.untitleddoc.cadencecalc.jaxrs.models.kotlin

import java.io.Serializable

public data class Perimeter(val etrto: String, val french: String, val perimeterValue: Int) : Serializable {
    public fun displayValue(): String = "$etrto ($french)"
}